### PR TITLE
Chore add admin report serializer

### PR DESCRIPTION
**Pull request by maciejSamerdak**
Introduces serializer for administrative purposes which will be used by both project-report-detail and admin-report-detail introduced in upcoming PRs.

**Do not delete this branch until #131 and #133 are merged**

### CODE REVIEW

File: employees/tests/test_unit_report_serializer.py [hunk 1]
```diff
@@ -132,6 +133,93 @@ def test_report_serializer_to_representation_method_should_replace_work_hours_wi
...
+    serializer_class = AdminReportSerializer
+
+    def test_report_serializer_creation_date_field_should_ignore_input_on_model_update(self):
+        timestamp = datetime.datetime(2001, 1, 1, 0, 0, 0, 0)
+        report = Report(
+            date=datetime.datetime.now().date(),
+            description=self.required_input["description"],
+            author=self.required_input["author"],
+            project=self.required_input["project"],
+            work_hours=Decimal("8.00"),
```
Comment (rwrzesien) [inline employees/tests/test_unit_report_serializer.py]: Why not `self.required_input["work_hours"]`?

File: employees/tests/test_unit_report_serializer.py [hunk 2]
```diff
@@ -132,6 +133,93 @@ def test_report_serializer_to_representation_method_should_replace_work_hours_wi
         self.assertEqual(data["work_hours"], "8:00")
 
 
+class AdminReportSerializerTests(DataSetUpToTests):
+    serializer_class = AdminReportSerializer
+
+    def test_report_serializer_creation_date_field_should_ignore_input_on_model_update(self):
+        timestamp = datetime.datetime(2001, 1, 1, 0, 0, 0, 0)
+        report = Report(
+            date=datetime.datetime.now().date(),
```
Comment (rwrzesien) [inline employees/tests/test_unit_report_serializer.py]: Please use `django.utils.timezone`.

File: employees/tests/test_unit_report_serializer.py [hunk 3]
```diff
@@ -132,6 +133,93 @@ def test_report_serializer_to_representation_method_should_replace_work_hours_wi
...
+        self.assertFalse(serializer.instance.creation_date is timestamp)
+
+    def test_report_serializer_last_update_field_should_ignore_input_on_model_update(self):
+        timestamp = datetime.datetime(2001, 1, 1, 0, 0, 0, 0)
+        report = Report(
+            date=datetime.datetime.now().date(),
+            description=self.required_input["description"],
+            author=self.required_input["author"],
+            project=self.required_input["project"],
+            work_hours=Decimal("8.00"),
```
Comment (rwrzesien) [inline employees/tests/test_unit_report_serializer.py]: Same here.

File: employees/tests/test_unit_report_serializer.py [hunk 4]
```diff
@@ -132,6 +133,93 @@ def test_report_serializer_to_representation_method_should_replace_work_hours_wi
...
+            },
+        )
+        self.assertTrue(serializer.is_valid())
+        serializer.save()
+        self.assertFalse(serializer.instance.creation_date is timestamp)
+
+    def test_report_serializer_last_update_field_should_ignore_input_on_model_update(self):
+        timestamp = datetime.datetime(2001, 1, 1, 0, 0, 0, 0)
+        report = Report(
+            date=datetime.datetime.now().date(),
```
Comment (rwrzesien) [inline employees/tests/test_unit_report_serializer.py]: Same here.

File: employees/tests/test_unit_report_serializer.py [hunk 5]
```diff
@@ -132,6 +133,93 @@ def test_report_serializer_to_representation_method_should_replace_work_hours_wi
...
+        serializer.save()
+        self.assertFalse(serializer.instance.last_update is timestamp)
+
+    def test_report_serializer_editable_field_should_ignore_input_on_model_update(self):
+        report = Report(
+            date=datetime.datetime.now().date(),
+            description=self.required_input["description"],
+            author=self.required_input["author"],
+            project=self.required_input["project"],
+            work_hours=Decimal("8.00"),
```
Comment (rwrzesien) [inline employees/tests/test_unit_report_serializer.py]: Same here.

File: employees/tests/test_unit_report_serializer.py [hunk 6]
```diff
@@ -132,6 +133,93 @@ def test_report_serializer_to_representation_method_should_replace_work_hours_wi
...
+                "last_update": timestamp,
+            },
+        )
+        self.assertTrue(serializer.is_valid())
+        serializer.save()
+        self.assertFalse(serializer.instance.last_update is timestamp)
+
+    def test_report_serializer_editable_field_should_ignore_input_on_model_update(self):
+        report = Report(
+            date=datetime.datetime.now().date(),
```
Comment (rwrzesien) [inline employees/tests/test_unit_report_serializer.py]: Same here.

**Comment by rwrzesien** [comment]
Looks good in general, some small improvements can be applied to tests. But the general question is - do we still need to add new serializers after we agreed to drop rest framework in favor of django views?

**Comment by kbeker** [comment]
This changes are unnecessary. I'm closing it

**Pull request closed without merging**